internal/publish/database/exposure_test.go

```diff
@@ -756,6 +756,10 @@ func TestIterateExposuresCursor(t *testing.T) {
 	if err == nil {
 		t.Fatal("expected error")
 	}
+	// IterateExposures doesn't guarantee the order, sort to make this test deterministic
```

Comment:
the preferred way to do this is to pass a sort function to cmp.Diff as a cmpopt

https://godoc.org/github.com/google/go-cmp/cmp/cmpopts#SortSlices

Reply:
Good to learn, done.